core/src/SYCL/Kokkos_SYCL_Instance.hpp

```diff
@@ -45,15 +45,19 @@
 #ifndef KOKKOS_SYCL_INSTANCE_HPP_
 #define KOKKOS_SYCL_INSTANCE_HPP_
 
-#include <Kokkos_SYCL.hpp>
+#include <Kokkos_Macros.hpp>
+
+#if defined KOKKOS_ENABLE_SYCL
```

Comment:
Why does this need to be guarded at all?

Reply:
Because it's included by `Kokkos_SYCL_Space.hpp`. Let me just guard it there instead.